src/main.rs

```diff
@@ -1283,13 +1310,57 @@ impl PofToolsGui {
                         }
                     }
 
-                    let size = 0.05 * model.sub_objects[obj_id].radius;
+                    let radius = model.sub_objects[obj_id].radius;
+                    let size = 0.05 * radius;
+                    let pos = model.get_total_subobj_offset(obj_id);
 
                     let mut lollipop_origin = GlLollipopsBuilder::new(LOLLIPOP_SELECTED_POINT_COLOR);
-                    lollipop_origin.push(model.get_total_subobj_offset(obj_id), Vec3d::ZERO, size);
+                    // Origin lollipop
+                    lollipop_origin.push(pos, Vec3d::ZERO, size);
                     let lollipop_origin = lollipop_origin.finish(display);
-
                     self.lollipops = vec![lollipop_origin];
+
+                    match model.sub_objects[obj_id].uvec_fvec() {
+                        Some((uvec, fvec)) => {
+                            // Set up arrowhead sticks
+                            let stick_length = 2. * radius;
+                            // Blue lollipop (stick only) for uvec
+                            let mut lollipop_uvec = GlLollipopsBuilder::new(LOLLIPOP_SELECTED_BANK_COLOR);
+                            lollipop_uvec.push(pos, uvec * stick_length, 0.);
+                            let lollipop_uvec = lollipop_uvec.finish(display);
+                            self.lollipops.push(lollipop_uvec);
+                            // Green lollipop (stick only) for fvec
+                            let fvec_colour = [0.15, 1.0, 0.15, 0.15];
```

Comment:
Make a new constant for this color. Additionally, add a new constant for your usage of `LOLLIPOP_SELECTED_BANK_COLOR` above, using the same value. We might want to change it later, after all.